types/cls-hooked/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for cls-hooked 4.2
+// Project: https://github.com/jeff-lewis/cls-hooked
+// Definitions by: Leo Liang <https://github.com/aleung>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { EventEmitter } from 'events';
+
+export interface NameSpace {
```

Comment:
This should be `Namespace` to match the casing of the other uses in this file

Reply:
@RyanCavanaugh Done.